### ISSUE TITLE
docs: unclear mention of `getIntrospectionQuery` in schema-awareness.md

### ISSUE THREAD

**Issue by Kingdutch** [FormidableLabs/urql#1228]
The [documentation mentions](https://github.com/FormidableLabs/urql/blame/main/docs/graphcache/schema-awareness.md#L110) that `getIntrospectionQuery` can take various arguments and is a convenience helper and unneeded in the example. However, the example doesn't provide it with any of the mentioned arguments (instead gives `{ description: false }`). 

The line in question is probably talking about another function. It probably means to say that `minifyIntrospectionQuery` takes many different arguments and that `getIntrospectedSchema` is there for convenience but isn't actually needed?

Given the example it's somewhat unclear what the paragraph is trying to explain.

**Comment by JoviDeCroock** [comment]
Hey,

This isn't really a bug so let's handle this in a discussion next time, you are indeed correct. We should rephrase this to talk about `minify` and that it's an optimization

**Comment by Kingdutch** [comment]
Ah sorry about that, I saw another docs issue so I wasn't sure where it qualified ^^

**Comment by kitten** [comment]
I think that's a typo and it's supposed to say `getIntrospectedSchema`
